### ISSUE TITLE
Video doesn't work on Firefox Windows

### ISSUE THREAD

**Issue by fsih** [LLK/scratch-gui#2216]
### Expected Behavior
When webcam access is allowed, video motion extension should show video in background of stage

### Actual Behavior

video not working

### Steps to Reproduce
1. Add video extension
2. Say yes on permissions
 Video doesn't appear and this error is in the console:
gui error Unhandled video io device error
MediaStreamError { name: "NotReadableError", message: "Failed to allocate videosource", constraint: "", stack: "" }

If you access the webcam from a different app on the computer, then try again in Firefox, then the video starts working.

### Operating System and Browser

Firefox Windows

**Comment by paulkaplan** [comment]
Yeah, we should handle this error and show a helpful message in this case.
